scripts/win32.bat

```diff
@@ -145,7 +148,7 @@ Function GetTopLevelDrives()
 				End If
 			Next
 
-			Summary.Add "Mountpoints", Mountpoints
+			Summary.Add "Mountpoints", MountpointObjects
```

Comment:
Perhaps the `"Mountpoints"` here (and on line 178) should be changed to `"MountpointObjects"` ?